voter-dapp/src/App.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./App.css";
+
+function App() {
+  return <div className="App">Voter DApp</div>;
```

Comment:
nit: I think we prefer the styling `dApp` for external viewing.

Reply:
Done.